static/app/components/events/eventCause.tsx

```diff
@@ -9,23 +9,25 @@ import {CauseHeader, DataSection} from 'sentry/components/events/styles';
 import {Panel} from 'sentry/components/panels';
 import {IconAdd, IconSubtract} from 'sentry/icons';
 import {t} from 'sentry/locale';
+import {
+  withCommitters,
+  WithCommittersProps,
```

Comment:
this name reminds me a lot of a `HoC`

Reply:
ah ok, because it's actually an `HoC` :D

Reply:
but in this case, shouldn't we get rid of the `withCommitters` and only use the context instead?

Reply:
No because this actually does the data loading. The context is the one who is responsible for storing it... Obviously, this is not ideal. withCommitters should be replaced by a useCommitters hook that does the same, but I didnt want to functionally change stuff, as it would imply migrating other components to fc's etc